common/CommunityToolkit.Labs.Shared/Renderers/ToolkitDocumentationRenderer.xaml.cs

```diff
@@ -213,4 +213,12 @@ private async void MarkdownTextBlock_LinkClicked(object sender, LinkClickedEvent
         }
     }
 #endif
+
+    public static Uri ToLabsUri(string path, int id) => new Uri($"https://github.com/CommunityToolkit/Labs-Windows/{path}/{id}");
```

Comment:
Let's remove "Labs" from this as well, for the same reason as above.

Reply:
Also, can we pull this from an environment variable? This can't stay hardcoded to the Labs repo.

Reply:
I'll see if I can pull from a prop in the csproj or something maybe? Or maybe a config file in the root that the sample heads reference? I'll investigate.

Reply:
I've cleaned-up the YAML front-matter label names. I'll open a 2nd PR for the url and the metadata reading as that's more complex (but close on the approach there, just need to debug some WASM stuff for Uno, but working in UWP/WinAppSDK now).